docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -483,19 +502,41 @@ Minimum compaction lag time is the minimum time a message remains uncompacted in
 [cols="25%,75%"]
 |===
 h|Editable
-|No
+|Yes
 
 h|Type
 |Long
 
 h|Default value
-|`0 seconds`
+|`0` (milliseconds)
 
 h|Kafka property name
 |`min.compaction.lag.ms`
 |===
 --
 
+Maximum compaction lag time::
++
+--
+Maximum compaction lag time is the maximum time a message remains uncompacted in a log. This property applies only when the topic cleanup policy is set to `Compact` or `Compact, Delete`.
+
+.Property attributes
+[cols="25%,75%"]
+|===
+h|Editable
+|Yes
+
+h|Type
+|Long
+
+h|Default value
+|`9223372036854775807` (milliseconds)
```

Comment:
Just checking that this value is actually correct? I used an online converter to check what it corresponds to, and it came out at, like, 290 million years? :D

Reply:
Ha! Yeah, I know. Yes, that's correct. It's the default Long value that kafka uses when it basically means "unlimited". You'll see this number come up all over the place in [kafka properties](https://kafka.apache.org/documentation/#topicconfigs).